keras/backend/tensorflow_backend.py

```diff
@@ -1163,6 +1163,30 @@ def cos(x):
     return tf.cos(x)
 
 
+def tan(x):
+    '''Computes tan of x element-wise.
```

Comment:
Yes, thank you! Wrt. docstrings: at the time of this writing, `tensorflow_backend.py` has docstrings for simple functions like `tan` while `theano_backend.py` has none. Do you think the goal is to have docstrings for all functions, or is it better if they are skipped for very simple functions?